src/inmanta/protocol/openapi/converter.py

```diff
@@ -169,9 +168,10 @@ def _handle_pydantic_model(self, type_annotation: Type, by_alias: bool = True) -
         # The ref_prefix changes the references, but the actual schemas are still at #/definitions
         schema = model_schema(type_annotation, by_alias=by_alias, ref_prefix=self.ref_prefix)
         if "definitions" in schema.keys():
-            definitions = schema.pop("definitions")
+            definitions: Dict[str, Dict[str, Any]] = schema.pop("definitions")
```

Comment:
```suggestion
            definitions: Dict[str, Dict[str, object]] = schema.pop("definitions")
```
Same comment here, no need for `Any` as far as I can see.

Reply:
I know it is not the best, but it matches the return types from `model_schema` (or actually `model_process_schema` and `model_type_schema` which are called my `model_schema`) and I was willing to stay consistent with that.

Reply:
The thing is, `object` is compatible with that because it's stricter. Actually anything would be compatible with it, you could just as well type it as `Dict[str, Dict[str, bool]]` even though that would be completely wrong. This sort of weak typing is exactly the reason why I'm of the opinion we should never use `Any` unless absolutely necessary (which is pretty rare). You're essentially saying to the type system "don't check me, I know best, I'm going to do whatever I feel like with this object and don't you tell me I can't". This defeats the purpose of typing for this part of the code. For this reason I'm of the opinion that when a method returns `Any`, we should always invert that assumption that "it can be anything and we can do whatever with it" to mean that "it can be anything so we can't do anything with it without making sure we actually can". In other words, always type it as `object`, because then the type system can stop you if you make assumptions.
Sorry for the rant, I realize the importance here is limited because we don't abuse the fact that it's `Any` at the moment. It's more that I'm trying to start a discussion about something I feel is important with regards to code quality in general. A strict type system can protect you from all kinds of errors in the code even before running it so why engage the off switch?

Reply:
Okay, I see your point then, I can not disagree with this.  I made the change then